tools/accuracy_checker/configs/open-closed-eye-0001.yml

```diff
@@ -0,0 +1,43 @@
+models:
+
+  - name: open-closed-eye-0001
+    launchers:
+      - framework: onnx_runtime
+        model: public/open-closed-eye-0001/open-closed-eye-0001.onnx
+        adapter: classification
+
+    datasets:
+      - name: mrlEyes_2018_01
+        preprocessing:
+          - type: resize
+            size: 32
+          - type: normalization
+            mean: 127.0, 127.0, 127.0
+            std: 255.0, 255.0, 255.0
+
+  - name: open-closed-eye-0001
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   public/open-closed-eye-0001/FP32/open-closed-eye-0001.xml
+        weights: public/open-closed-eye-0001/FP32/open-closed-eye-0001.bin
+        adapter: classification
+
+      - framework: dlsdk
+        tags:
+          - FP16
+        model:   public/open-closed-eye-0001/FP16/open-closed-eye-0001.xml
+        weights: public/open-closed-eye-0001/FP16/open-closed-eye-0001.bin
+        adapter: classification
+
+    datasets:
+      - name: mrlEyes_2018_01
+        preprocessing:
+          - type: resize
+            size: 32
+          - type: normalization
+            mean: 127.0, 127.0, 127.0
+            std: 255.0, 255.0, 255.0
+
```

Comment:
@ygnn123  it is common practice that for running model on IE we do not provide mormalization in AC config, because it is a part of model after MO conversion, so normalization will be applied twice if you do not remove it here and we get incorrect results.